minutes/board_2019_02_06.md

```diff
@@ -0,0 +1,46 @@
+# NiT Meeting 02/06/2020
+
+### Agenda
+
+### In Attendance
+
+* Adam Recvlohe
+* Coty Sutherland
+* Nick Sahler
+
+### Update
+
+* Adam
+    * Received $60 more dollars from Benevity Causes
+    * Worked on updating documentation for our projects on GitHub
+    * Met with Diana and came up with questions to ask potential candidates to Coding Dojo Program
+    * Emailed Mark Smith from Coding Dojo about starting the program. I will email him back about a possible starting date
+    * Add Netlify logo to NiT website in order to keep using Netlify resources for free
+    * A few people are helping to work on the NiT website but things have slowed down
+    * Let conference volunteers know we are doing another remote conference
+    * Could not get everyone’s id :(
+    * I will hopefully speak at AISES Region 4 Conference (I should probably have a table too)
+    * College of Muscogee Nation is having a higher ed conference at River Spirit Casino which I will table at
+    * Meeting with Alex on Saturday at noon
+    * Started using buffer.io to send social media updates to all three platforms (twitter, fb, and linkedin)
+* Coty
+    * Still have some money in my D+I community budget. If we have any events or anything I may be able to sponsor them.
+    * Not much work done for NiT, but I’m pretty busy with my D+I work and VP responsibilities for TNAS.
+    * May have a table for $work at the AISES Region 7 conference in March
```

Comment:
What is `$work`?

Reply:
Maybe this is a placeholder for Coty's actual employer. Can we replace this with some generic phrase instead of using variable semantics?
`May have a table for Coty's employer at the AISES Region 7 conference in March`

Reply:
Yeah, $work or $dayjob is a common placeholder used in various mailing lists that I'm on. I'll fix it.